rules-reviewed/jboss-eap/websphere/xml-websphere.windup.xml

```diff
@@ -1,22 +1,35 @@
 <?xml version="1.0"?>
-<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="XmlWebsphereRules"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+<ruleset id="xml-websphere" xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of WebSphere proprietary xml files and constructs that may require individual attention when migrating to JBoss EAP 6+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,2.3.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="websphere" />
+        <targetTechnology id="eap" versionRange="[6,)" />
+    </metadata>
     <rules>
-        <rule id="XmlWebsphereRules_2fmb">
+        <rule id="xml-websphere-01000">
             <when>
-                <xmlfile as="default" matches="/rl:RuleSet">
+                <xmlfile matches="/rl:RuleSet">
                     <namespace prefix="rl" uri="http://www.ibm.com/xmlns/prod/websphere/wbi/br/6.0.0"/>
                 </xmlfile>
             </when>
             <perform>
                 <iteration>
-                    <classification title="IBM Process Server Rules 6.0" effort="0"/>
+                    <classification title="IBM Process Server Rules 6.0" effort="3">
+			<description>WBI is a websphere specific business integration solution. This needs to be migrated to Drools.</description>
+			<link href="TODO" description="WBI to Drools migration"/>
```

Comment:
I think that it would be better to have a JIRA on the documentation than to have a TODO link in the rules. I don't think that we can merge it with TODO links like this.